### DISCUSSION TITLE
Are identificationType and identificationNumber mandatory to create a card token?

### DISCUSSION THREAD

**Discussion by Oteiza-a** [mercadopago/sdk-js#166] [General]
I am using the `mpinstance.fields.createCardToken` function, from the Core Methods to generate the card token to save that card and then generate the payment, and i am using it as seen in the Core Methods docs:

```
const token = await mp.fields.createCardToken({
  cardholderName: document.getElementById('form-checkout__cardholderName').value,
  identificationType: document.getElementById('form-checkout__identificationType').value,
  identificationNumber: document.getElementById('form-checkout__identificationNumber').value,
});
```

But i was wondering if the `identificationType` and `identificationNumber` are mandatory, because i was actually able to do it by just sending the `cardholderName`, but maybe that could have negative repercussions in the future, that's why i am making this question.

As for why i am asking, it's just because it would improve the UX of the frontend to have less form fields that the user has to fill to 
make the payment.

Thanks in advance

**Comment by lucmantovani** [comment]
Hello @Oteiza-a!

In the result of mp.getPaymentMethods there is a field called `additional_info_needed`
The identificationType and identificationNumber are mandatory will appear inside this property when mandatory (but from experience, usually they are and filling them helps in payment approval).

You can see more about accessing this property in the issuer part from the documentation: https://www.mercadopago.com.br/developers/en/docs/checkout-api/integration-configuration/card/integrate-via-core-methods#bookmark_get_issuing_bank.

Also, it seems a nice opportunity to inform you that further questions related on integrating our products are best addressed by contacting support directly: https://www.mercadopago.com/developers/en/support/center/tickets

Hope it works out to you! 🙂

**Reply by Oteiza-a** [reply]
Thank you @lucmantovani, we will keep the fields then.
